src/main/java/de/uniwue/web/io/PageXMLReader.java

```diff
@@ -217,4 +220,26 @@ public static PageAnnotations loadPageAnnotationsFromDisc(File pageXMLInputPath)
 
 		return segResult;
 	}
+
+	/**
+	 * Read the Orientation of a PageXML Document
+	 *
+	 * @param pagexml Path to the PageXML file
+	 * @return orientation skew angle of page element
+	 */
+	public static double getPageOrientation(Document pagexml) {
```

Comment:
We should use prima-core-libs whenever possible so I propose replacing this function with
```Java
public static double getOrientation(Page page){
	if(page.getAttributes().get("orientation") != null && page.getAttributes().get("orientation").getValue() != null){
		return Double.parseDouble(page.getAttributes().get("orientation").getValue().toString());
	}
	return 0.0;
}
```